pwiz_tools/Skyline/TestFunctional/DdaSearchTest.cs

```diff
@@ -45,8 +45,9 @@ namespace pwiz.SkylineTestFunctional
     [TestClass]
     public class DdaSearchTest : AbstractFunctionalTest
     {
-        public struct ExpectedResults
+        public class ExpectedResults
         {
+            public Exception expectedException;

```

Comment:
I guess we can leave this for consistency, but it is non-standard naming to have camel-case public member variables. Our standard style is title-case for all of the public interface. We use camel-case preceded by an underscore for private member variables:

private string _privateMember;

Reply:
Yes, might change all those to title case at some point. Probably not this PR since it'd just add noise.